tools/pip_install.py

```diff
@@ -47,10 +48,8 @@ def certbot_normal_processing(tools_path, test_constraints):
     with open(certbot_requirements, 'r') as fd:
         data = fd.readlines()
     with open(test_constraints, 'w') as fd:
-        for line in data:
-            search = re.search(r'^(\S*==\S*).*$', line)
-            if search:
-                fd.write('{0}{1}'.format(search.group(1), os.linesep))
+        data = os.linesep.join(strip_hashes.process_entries(data))
```

Comment:
nit: Python docs say [not to use os.linesep when writing to text files](https://docs.python.org/3/library/os.html#os.linesep).